File: workspace/Plugins/bridges/JRuby/plotDstIndex2.rb
#!/usr/bin/env jruby
include Java
require "/home/iugonet/Judas/lib/external/jcommon-1.0.17.jar"
require "/home/iugonet/Judas/lib/external/jfreechart-1.0.14.jar"
require "/home/iugonet/Judas/lib/Aplot.jar"
require "/home/iugonet/Judas/lib/Tplot.jar"
require "/home/iugonet/Judas/plugins/DstIndex.jar"
#require_relative "/home/iugonet/Judas/lib/external/jcommon-1.0.17.jar"
#require_relative "/home/iugonet/Judas/lib/external/jfreechart-1.0.14.jar"
#require_relative "/home/iugonet/Judas/lib/Aplot.jar"
#require_relative "/home/iugonet/Judas/lib/Tplot.jar"
#require_relative "/home/iugonet/Judas/plugins/DstIndex.jar"

java_import org.jfree.chart.ChartFactory
java_import org.jfree.chart.ChartFrame
java_import java.lang.String
java_import java.lang.Boolean
java_import org.iugonet.www.DstIndex

strUrl="http://wdc-data.iugonet.org/data/hour/index/dst/1984/dst8410"
dstIndex=DstIndex.new()
timeSeriesCollection=dstIndex.loadData(strUrl)

chart=ChartFactory.createTimeSeriesChart(String("Dst Index"),String("Time"),String("Dst Index [nT]"),timeSeriesCollection,FALSE,FALSE,TRUE)
frame=ChartFrame.new(String("Simple TimeSeries Chart"),chart)
frame.pack()
frame.setVisible(true)

File: workspace/Plugins/bridges/Jython/plotDstIndex2.py
#!/usr/bin/env jython
import sys

# Add jar path
sys.path.append("/home/iugonet/Judas/lib/Tplot.jar")
sys.path.append("/home/iugonet/Judas/lib/Aplot.jar")
sys.path.append("/home/iugonet/Judas/lib/external/jcommon-1.0.17.jar")
sys.path.append("/home/iugonet/Judas/lib/external/jfreechart-1.0.14.jar")
sys.path.append("/home/iugonet/Judas/plugins/DstIndex.jar")

from org import jfree
from org.iugonet.www import Aplot
from org.iugonet.www import Tplot
from org.iugonet.www import DstIndex
from java import lang

strUrl="http://wdc-data.iugonet.org/data/hour/index/dst/1984/dst8410"
dstIndex=DstIndex()
timeSeriesCollection=dstIndex.loadData(strUrl)

chart=jfree.chart.ChartFactory.createTimeSeriesChart(lang.String("Dst Index"),lang.String("Time"),lang.String("Dst Index [nT]"),timeSeriesCollection,lang.Boolean("FALSE"),lang.Boolean("FALSE"),lang.Boolean("TRUE"))
frame=jfree.chart.ChartFrame(lang.String("Simple TimeSeries Chart"),chart)
frame.pack()
frame.setVisible(lang.Boolean("TRUE"))
